src/main/java/io/quarkus/gizmo2/creator/ops/OptionalOps.java

```diff
@@ -0,0 +1,59 @@
+package io.quarkus.gizmo2.creator.ops;
+
+import java.util.Optional;
+
+import io.quarkus.gizmo2.Expr;
+import io.quarkus.gizmo2.creator.BlockCreator;
+
+/**
+ * Operations on {@link Optional}.
+ */
+public class OptionalOps extends ObjectOps {
+
+    /**
+     * Construct a new instance.
+     * 
+     * @param bc
+     * @param obj
+     */
+    public OptionalOps(BlockCreator bc, Expr obj) {
+        super(Optional.class, bc, obj);
+    }
+
+    /**
+     * Generate a call to {@link Optional#get()}.
+     *
+     * @return the expression of the result (not {@code null})
+     */
+    public Expr get() {
+        return invokeInstance(Object.class, "get");
+    }
+
+    /**
+     * Generate a call to {@link Optional#get()}.
+     *
+     * @return the expression of the result (not {@code null})
+     */
+    public Expr isPresent() {
+        return invokeInstance(boolean.class, "isPresent");
+    }
+
+    /**
+     * Generate a call to {@link Optional#get()}.
+     *
+     * @return the expression of the result (not {@code null})
+     */
+    public Expr isEmpty() {
+        return invokeInstance(boolean.class, "isEmpty");
+    }
+
+    /**
+     * Generate a call to {@link Optional#get()}.
+     *
+     * @return the expression of the result (not {@code null})
```

Comment:
Missed the doc for parameter `other` here.

Reply:
Done.